mathics/builtin/messages.py

```diff
@@ -566,16 +566,16 @@ class Syntax(Builtin):
     </dl>
 
     >> 1 +
-     : Incomplete expression; more input is needed (line 1 of "<test>").
+     : Incomplete expression; more input is needed (line 1 of "").
```

Comment:
So now things like `<test>` are ignored?  Where in the code is this done?

I don't see this aspect described in https://mathics-development-guide.readthedocs.io/en/latest/extending/developing-code/extending/documentation-markup.html

Reply:
In the current master docpipeline, line 120, it is defined a  `MathicsSingleLineFeeder` that feeds the parser and identifies itself as `<test>`. Then, if a syntax error happens, the error shows this kind of error finished by `'(line 1 of "<test>")'`. 
In this branch, to make the test system more homogeneous, I propose to use a MathicsSession object which has its own parser (see Line 91 from `mathics.session`).
I can reestablish the "<test>" by adding it in the `MathicsSession.evaluation` method, or I can make it customizable.

Reply:
Ok thanks for the information.

Somehow I think practical information about what to do about file locations should be conveyed to a would-be doctest writer. I am not sure I would know what to say though.